### PR TITLE
[IMP] hr: unify the plan summary

### PR DESCRIPTION
**Pull request by amah-odoo**
- edit the codition to check if the active model is HR to use the smae plan summary as the buttom rfom the employee from

Task: 4237311

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr/wizard/mail_activity_schedule.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def _compute_plan_department_filterable(self):
 
     @api.depends('plan_date', 'plan_id')
     def _compute_plan_summary(self):
-        if not self.env.context.get('sort_by_responsible', False):
+        if not self.env.context.get('sort_by_responsible', False) and self.env.context.get('active_model', False) != 'hr.employee':
```
Comment (tivisse) [inline addons/hr/wizard/mail_activity_schedule.py:34]: What is the purpose of this change ? It seems like a fix, it could be isolated into a separate commit then. 

Thanks for your work !

**Review by tivisse — COMMENTED** [review]
Small question :)

File: addons/hr/wizard/mail_activity_schedule.py (hunk 1 above)
Comment (amah-odoo) [reply addons/hr/wizard/mail_activity_schedule.py:34]: Because the button inside the form call the action with this context key but the activity button inside the kanban view does not and if we inherited the Kanban view one this would affect all other kanban views So I have made the condition to check if this call happen inside the hr.employee module.
I didn't remove the context to let it available for other modules to call the action with it if needed

**Comment by tivisse** [comment]
@robodoo r+
